airbyte-integrations/connectors/source-monday/source_monday/dpath_string_extractor.py

```diff
@@ -12,11 +12,10 @@
 from airbyte_cdk.sources.declarative.extractors.record_extractor import RecordExtractor
 from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
 from airbyte_cdk.sources.declarative.types import Config, Record
-from dataclasses_jsonschema import JsonSchemaMixin
 
 
 @dataclass
-class DpathStringExtractor(RecordExtractor, JsonSchemaMixin):
+class DpathStringExtractor(RecordExtractor):
```

Comment:
We can now remove this class.
https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/record-selector#selecting-fields-nested-in-arrays